packages/fleather/lib/src/widgets/editor.dart

```diff
@@ -1013,10 +1006,12 @@ class RawEditorState extends EditorState
 
     if (widget.focusNode != oldWidget.focusNode) {
       oldWidget.focusNode?.removeListener(_handleFocusChanged);
-      _focusNode?.removeListener(_handleFocusChanged);
-      _focusNode?.dispose();
-      _focusNode = null;
-      _createInternalFocusNodeIfNeeded();
+      if (widget.focusNode != null) {
+        _focusNode
+          ?..removeListener(_handleFocusChanged)
+          ..dispose();
```

Comment:
maybe add a comment somewhere to explain _focusNode is a potential internal focusNode? (or name it _internalFocusNode?)

Just to make it clear that it has to be handled only when we created it because a user one was not provided.

Reply:
Done. @cgestes